tox.ini

```diff
@@ -134,7 +134,7 @@ basepython = python3
 description =
   Build package, verify metadata, install package and assert behavior when ansible is missing.
 deps =
-  pep517 >= 0.7.0
+  build >= 0.6.0, < 0.7.0
```

Comment:
I would personally do not bother to put a cap there. In fact I would prefer to findout if newer versions breaks CI instead of having someone raise a bug for it.

Reply:
For now, I prefer stability here. There are enough end-users of build to report issues upstream.